test/support/fixtures_support.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
+  module FixturesSupport
+    # rubocop:disable Style/AccessorMethodName
+    def self.set_fixture_classes(test_class)
+      test_class.set_fixture_class :katello_activation_keys => "Katello::ActivationKey"
+      test_class.set_fixture_class :katello_content_views => "Katello::ContentView"
+      test_class.set_fixture_class :katello_content_view_environments => "Katello::ContentViewEnvironment"
+      test_class.set_fixture_class :katello_content_view_filters => "Katello::ContentViewFilter"
+      test_class.set_fixture_class :katello_content_view_erratum_filter_rules => "Katello::ContentViewErratumFilterRule"
+      test_class.set_fixture_class :katello_content_view_package_filter_rules => "Katello::ContentViewPackageFilterRule"
+      test_class.set_fixture_class :katello_content_view_package_group_filter_rules => "Katello::ContentViewPackageGroupFilterRule"
+      test_class.set_fixture_class :katello_content_view_puppet_modules => "Katello::ContentViewPuppetModule"
+      test_class.set_fixture_class :katello_content_view_puppet_environments => "Katello::ContentViewPuppetEnvironment"
+      test_class.set_fixture_class :katello_content_view_repositories => "Katello::ContentViewRepository"
+      test_class.set_fixture_class :katello_content_view_version_environments => "Katello::ContentViewVersionEnvironment"
+      test_class.set_fixture_class :katello_content_view_versions => "Katello::ContentViewVersion"
+      test_class.set_fixture_class :katello_distributors => "Katello::Distributor"
+      test_class.set_fixture_class :katello_environment_priors => "Katello::EnvironmentPrior"
+      test_class.set_fixture_class :katello_environments => "Katello::KTEnvironment"
+      test_class.set_fixture_class :katello_gpg_keys => "Katello::GpgKey"
+      test_class.set_fixture_class :katello_products => "Katello::Product"
+      test_class.set_fixture_class :katello_providers => "Katello::Provider"
+      test_class.set_fixture_class :katello_repositories => "Katello::Repository"
+      test_class.set_fixture_class :katello_sync_plans => "Katello::SyncPlan"
+      test_class.set_fixture_class :katello_host_collections => "Katello::HostCollection"
+      test_class.set_fixture_class :katello_systems => "Katello::System"
+      test_class.set_fixture_class :katello_system_host_collections => "Katello::SystemHostCollection"
+      test_class.set_fixture_class :katello_task_statuses => "Katello::TaskStatus"
+      test_class.set_fixture_class :katello_errata => "Katello::Erratum"
+      test_class.set_fixture_class :katello_erratum_packages => "Katello::ErratumPackage"
+      test_class.set_fixture_class :katello_erratum_cves => "Katello::ErratumCve"
+      test_class.set_fixture_class :katello_repository_errata => "Katello::RepositoryErratum"
+      test_class.set_fixture_class :katello_system_errata => "Katello::SystemErratum"
+    end
```

Comment:
Would it make more sense for this to just be a hash, and then in the calling function it could just do something like:
```
FixtureSupport::FIXTURE_MAPPING.each do |key,value|
   self.set_fixture_class(key => value)
end
```